### PR TITLE
Student form

### PR DESCRIPTION
**Pull request by hpetru**

### CODE REVIEW

File: app/admin/student.rb [hunk 1]
```diff
@@ -30,4 +35,53 @@
...
+        input_html: { placeholder: '2016-01-01' }
+      f.input :foreign_language
+      f.input :student_group, as: :select,
+        include_blank: false,
+        input_html: { class: 'select2able' }
+    end
+
+    f.inputs 'Adresa' do
+      f.input :address,
+        label_method: :display_name,
```
Comment (doringutsu) [inline app/admin/student.rb:82]: I agree

File: app/admin/student.rb [hunk 2]
```diff
@@ -30,4 +35,53 @@
...
+      f.input :gender, as: :select,
+        include_blank: false
+      f.input :idnp_token
+      f.input :nationality, as: :select,
+        include_blank: false
+    end
+
+    f.inputs 'Informație clasă' do
+      f.input :came_from,
+        input_html: { class: 'select2able' }
```
Comment (doringutsu) [inline app/admin/student.rb:71]: You should definetely do that!

Comment (hpetru) [inline app/admin/student.rb:71]: Inchidi gura, @doringutsu @houndci-bot

File: app/admin/student.rb (hunk 1 above)
Comment (hpetru) [inline app/admin/student.rb:82]: Gura

**Pull request merged**